openlibrary/templates/work_search.html

```diff
@@ -76,7 +76,7 @@
         \$("#" + header + "_more").show();
         \$("#" + header + "_bull").show();
     }
-    \$(facetEntry + ":not(:hidden)").slice(shown-facet_inc,shown).addClass('hidden');
+    \$(facetEntry + ":not(:hidden)").slice(shown-facet_inc,shown).addClass('hidden-2');
 }
 window.q.push(function(){
     \$(".header_bull").hide();
```

Comment:
```suggestion
    \$(".header_bull").hide();
    \$(".facet-less").hide();
```

Reply:
The real issue is that the `.hidden` class is never removed from the "less" link.  Adding a new class to the link and hiding them all when the page loads solves this issue without the need for new `.hidden` classes.

Reply:
@jamesachamp Done. Please take a look